src/blind.rs

```diff
@@ -447,10 +448,106 @@ impl TxOut {
 
         Ok((txout, out_abf, out_vbf))
     }
+
+    /// Unblinds a transaction output, if it is confidential.
+    ///
+    /// It returns the secret elements of the value and asset Pedersen commitments.
+    pub fn unblind<C: Verification>(
+        &self,
+        secp: &Secp256k1<C>,
+        blinding_key: SecretKey,
```

Comment:
I think now (since #82) this should be a `ValueBlindingFactor`

Reply:
I think value/asset blinding factors are different to the blinding key used in combination with the nonce to unblind the output. In fact, the value blinding factor of the output is returned as part of unblinding it.

Perhaps you are referring to the fact that blinding factors are now modelled as `Tweak`s in `secp256k1-zkp`. I'm not sure if the "blinding key" used to unblind the output deserves the same treatment, or if we can still model it as a secret key.

Reply:
You're right, I was confused by our poor terminology here.